pkg/app/AppListingViewBuilder.go

```diff
@@ -60,12 +61,16 @@ func (impl *AppListingViewBuilderImpl) BuildView(fetchAppListingRequest FetchApp
 
 	var appContainersResponses []*bean.AppContainer
 	for k, v := range filteredAppEnvMap {
-		appId, err := strconv.Atoi(strings.Split(k, "_")[0])
+		appIdAndName := strings.Split(k, "_")
```

Comment:
this logic is for finding  ACD name, there is a common utility where we are abstracting this logic please use that

Reply:
@iamayushm please help here

Reply:
@nishant-d I checked with @iamayushm , function to get ACD name is where we get app name and env name and split on basis of hyphen (-), here need to get appId, and appName and separater is underscore.